src/components/menus/ContextMenu/ContextMenu.tsx

```diff
@@ -50,15 +50,14 @@ const ContextMenu = (props: IContextMenuProps) => {
                                 onClick={(event) => onClickItem(event, item)}
                                 disabled={item.enabled === undefined ? false : !item.enabled}
                                 intent={item.color === 'red' ? 'destructive' : 'default'}
-                                style={{ flexGrow: 1, padding: '5px 0', lineHeight: '20px' }}
-                                privateOptions={{ fontWeight: 'medium' }}
+                                style={{ flexGrow: 1, padding: '5px 0' }}
```

Comment:
Thank you for this change... I had spotted this previously and was agitated LOL

Reply:
Yeah, it was all of the sudden included in the designs, but Tyson said we should probably just remove it, thankfully!